src/Symfony/Component/Mailer/Command/MailerDebugCommand.php

```diff
@@ -0,0 +1,137 @@
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * Helps making sure your mailer provider is operational.
+ *
+ * @author Guillaume MOREL <me@gmorel.io>
+ *
+ * @final
+ */
+class MailerDebugCommand extends Command
+{
+    /** {@inheritdoc} */
+    protected static $defaultName = 'debug:mailer:send-email';
+
+    /** @var ?MailerInterface */
+    private $mailer;
+
+    /** @var SymfonyStyle */
+    private $io;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(?MailerInterface $mailer)
+    {
+        parent::__construct();
+
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isEnabled()
+    {
+        return null !== $this->mailer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Send simple email message')
+            ->setDefinition([
+                new InputOption('from', null, InputOption::VALUE_REQUIRED, 'The from email of the message'),
+                new InputOption('to', null, InputOption::VALUE_REQUIRED, 'The to email of the message'),
+                new InputOption('subject', null, InputOption::VALUE_REQUIRED, 'The subject of the message'),
+                new InputOption('body', null, InputOption::VALUE_REQUIRED, 'The body of the message'),
+            ])
+            ->setHelp(
+                <<<EOF
+The <info>%command.name%</info> command creates and sends a simple email message.
+Usage:
+- <info>php %command.full_name%</info> will trigger the interactive mode
+- <info>php %command.full_name% --from=a@symfony.com --to=b@symfony.com --subject=Test --body=body</info>
+EOF
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws TransportExceptionInterface
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $email = $this->createEmail($input);
+
+        try {
+            $this->mailer->send($email);
+        } catch (TransportExceptionInterface $e) {
+            $this->io->error('Fail to send email.');
+            throw $e;
+        }
+
+        $this->io->success(
+            sprintf(
+                'Email was successfully sent to "%s".',
+                (string) $input->getOption('to')
+            )
+        );
+
+        return Command::SUCCESS;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        foreach ($input->getOptions() as $option => $value) {
+            if (null === $value) {
+                $input->setOption($option, $this->io->ask(sprintf('%s', ucfirst($option))));
```

Comment:
IMHO Subject and Message could be optional. The purpose is to test the setup. sending a dummy message is enough is 99% of cases.
Same for `from`. Only the `to` is required isn't it?

Reply:
I agree
I will try to come up with something mergeable by the weekend